src/routes/safe/components/Transactions/TxList/BatchExecuteHoverProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import { createContext, ReactElement, ReactNode, useState } from 'react'
+
+export const BatchExecuteHoverContext = createContext<{
+  activeHover?: string[]
+  setActiveHover: (activeHover?: string[]) => void
+}>({
+  activeHover: undefined,
+  setActiveHover: () => {},
+})
+
+export const BatchExecuteHoverProvider = ({ children }: { children: ReactNode }): ReactElement => {
```

Comment:
Please add a comment here as to why this is needed as we already have a hover provider.